### PR TITLE
Fix missing OutputTimestamp property

### PR DESCRIPTION
**Pull request by charphi**
Fix problem 1 in #133 

I had to add the maven-archiver dependency to get the timestamp parsing code. I hope this doesn't break anything.
I also found the timestamp problem to be present in some other mojo.

### CODE REVIEW

**Comment by ebarboni** [comment]
just as it's first contribution you should take sometimes to read this (not all things apply) and reply saying you are ok with Apache code of conduct

 https://netbeans.apache.org/front/main/participate/submit-pr/#contributing-code
 
 PR Looks good to me

**Review by mbien — APPROVED** [review]
looks good to me! Thanks for the contribution.

File: nbm-maven-plugin/src/main/java/org/apache/netbeans/nbm/AbstractNbmMojo.java [hunk 1]
```diff
@@ -23,10 +23,9 @@
 import java.io.FileReader;
 import java.io.IOException;
 import java.io.Reader;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+
```
Comment (ebarboni) [inline nbm-maven-plugin/src/main/java/org/apache/netbeans/nbm/AbstractNbmMojo.java:27]: avoid star import *

**Comment by charphi** [comment]
> just as it's first contribution you should take sometimes to read this (not all things apply) and reply saying you are ok with Apache code of conduct
> 
> https://netbeans.apache.org/front/main/participate/submit-pr/#contributing-code
> 
> PR Looks good to me

I'm ok with the Apache code of conduct

File: nbm-maven-plugin/src/main/java/org/apache/netbeans/nbm/AbstractNbmMojo.java (hunk 1 above)
Comment (charphi) [reply nbm-maven-plugin/src/main/java/org/apache/netbeans/nbm/AbstractNbmMojo.java]: I've just fixed it

**Comment by ebarboni** [comment]
if you could rebase in one commit would be perfect :D

**Comment by charphi** [comment]
Do you mean squashing commits? Can it be done if commits are already in the PR?

**Comment by mbien** [comment]
> Do you mean squashing commits? Can it be done if commits are already in the PR?

you can squash locally and then force push with `-f` into the PR - would be great so that we can merge it as-is.

**Comment by charphi** [comment]
I just squashed and force-pushed the commit. It seems to work properly.

I also just found out that github has a similar possibility : https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-commits
Have you tried it? Does it behave the same way?

**Comment by mbien** [comment]
@charphi cool thanks. Yes we know that we can let github squash during integration. This caused problems in past, author information can get lost etc and it add opportunities for more mistakes, e.g selecting the wrong option and then merging 20 commits instead of squashing - which happened too.

We simply ask squash the PR to the desired number of commits as last step post review. We can then merge the commits unmodified without changing anything.

**Comment by ebarboni** [comment]
thks for the contribution

**Pull request merged**